### ISSUE TITLE
Add a guide about usage of sessions

### ISSUE THREAD

**Issue by Barbapapazes** [nitrojs/nitro#1444]
### Describe the feature

I think that we could add a guide about usage of sessions (same approach that I've done in nuxt/nuxt#1307). 

### Additional information

- [X] Would you be willing to help implement this feature?

**Comment by Hebilicious** [comment]
I think this absolutely needs to be defined somewhere.
Ideally, this should be in a section of a H3 website, and both the Nuxt and Nitro sites should have H3 as a "subsite"
https://github.com/nuxt/nuxt/issues/21930

**Comment by Barbapapazes** [comment]
> I think this absolutely needs to be defined somewhere.
> Ideally, this should be in a section of a H3 website, and both the Nuxt and Nitro sites should have H3 as a "subsite"
> https://github.com/nuxt/nuxt/issues/21930

Yes, I'm ok for the h3 website but currently there is not documentation for this website!

**Comment by pi0** [comment]
This is a good idea. Setting up h3 docs soon. I think we can document it in both places because nitro docs can include more details such as configuration and using runtime config (in the future I'm thinking to make it even a built-in feature for easy usage)
